server/src/modules/user/user.service.ts

```diff
@@ -102,6 +107,41 @@ export class UserService {
     }
   }
 
+  async getUserById(id: string): Promise<User> {
+    try {
+      return await this.userModel.findById(id, {
```

Comment:
```suggestion
      return this.userModel.findById(id, {
```
await with return statement not needed